packages/cozy-authentication/src/MobileRouter.jsx

```diff
@@ -69,15 +72,32 @@ export class MobileRouter extends Component {
     client.removeListener('logout', this.afterLogout)
   }
 
+  startListeningToUniversalLinks() {
+    if (window.universalLinks) {
+      window.universalLinks.subscribe(
+        'openUniversalLink',
+        this.startHandlingUniversalLinks
+      )
+    }
+  }
+
   componentWillUnmount() {
     this.stopHandlingDeeplinks()
     this.stopListeningToClient()
+    this.stopListeningToUniversalLinks()
     this.unmounted = true
   }
 
   update() {
     this.forceUpdate()
   }
+  startHandlingUniversalLinks(eventData) {
+    /* 
+   openUniversalLink seems to be called only on iOS.
+   android uses handleOpenURL by default ?!
+   */
```

Comment:
I find this comment not very clear. Is it to explain something ? Is it a TODO ? Please clarify

Reply:
it's a todo. I need to double check the behavior on Android.